doc/uftrace-live.md

```diff
@@ -254,8 +254,8 @@ filter; an opt-in filter with `-F`/`--filter` and an opt-out filter with
 replay time.
 
 The first one is an opt-in filter. By default, it doesn't trace anything.  But
-when one of the specified functions is executed, tracing is started.  When the
-function returns, tracing is stopped again.
+when one of the specified functions is executed, tracing will be started.  When the
+function returns, tracing will be stopped again.
```

Comment:
It seems that it doesn't have to be changed. Please correct me if I'm wrong.

Reply:
I agree with you. Restore fix done.